blog/18-04-2023-interacting-with-celo-stablecoin-celo-dollars-using-python/interacting-with-celo-stablecoin-celo-dollars-using-python.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Interacting with Celo's Stablecoin, Celo Dollars, using Python
+description: Uncover the secrets of working with Celo's stablecoin, Celo Dollars, in this informative guide. Learn how to interact, manage, and utilize Celo Dollars in your applications using Python. This article covers essential concepts, practical examples, and step-by-step instructions to harness the power of Celo's blockchain technology, opening up new possibilities in the world of digital currencies and decentralized finance.
+authors:
+  - name: Joshua Moses
+url: https://github.com/Divine572
+tags: [celosage, intermediate, smartcontract, solidity]
```

Comment:
can you add a python tag here as well that;ll be all

Reply:
actually the python tag isnt valid so why don't you remove the one in your user.tsx file instead